### PR TITLE
Adding AttrDict get method.

### LINKED ISSUE #90: [FEATURE] AttrDict get(...) method
### Is your feature request related to a problem?
_just like dict, AttrDict could have a get method._

```
In [1]: from opensearch_dsl import AttrDict
In [2]: a = AttrDict({"a": "b"})
In [3]: a
Out[3]: {'a': 'b'}
In [4]: a.a
Out[4]: 'b'
In [5]: a["a"]
Out[5]: 'b'
In [6]: a.get("a", default="c")
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
File ...venv/lib/python3.8/site-packages/opensearch_dsl/utils.py:168, in AttrDict.__getattr__(self, attr_name)
    167 try:
--> 168     return self.__getitem__(attr_name)
    169 except KeyError:

File .../venv/lib/python3.8/site-packages/opensearch_dsl/utils.py:187, in AttrDict.__getitem__(self, key)
    186 def __getitem__(self, key):
--> 187     return _wrap(self._d_[key])
```

### What solution would you like?
_Same behaviour as dict._

### PR DESCRIPTION
**Pull request by jonatasalves-hotmart**
Signed off by: jonatas.alves@hotmart.com

### Description
Adding get(...) method in AttrDict

### Issues Resolved
#90 

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass.
- [ ] New functionality has been documented.
  - [ ] New functionality has comments added.
- [x] Commits are signed per the DCO using --signoff.
- [x] [CHANGELOG](https://github.com/opensearch-project/opensearch-dsl-py/blob/main/CONTRIBUTING.md#changelog) has been updated.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following the Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/opensearch-dsl-py/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -109,3 +109,13 @@ def test_recursive_to_dict():
...
+
+
+def test_attrdict_get():
+    a = utils.AttrDict({"a": {"b": 42, "c": 47}})
+    assert a.get("a", {}).get("b", 0) == 42
+    assert a.get("a", {}).get("e", 0) == 0
+    assert a.get("d", {}) == {}
+    with raises(AttributeError):
+        assert a.get("d")
+
```
Comment (Yury-Fridlyand) [inline tests/test_utils.py:121]: I suppose lint requires an empty line here:
```suggestion

```

**Review by Yury-Fridlyand — COMMENTED** [review]
Please, update `CHANGELOG.md`

File: tests/test_utils.py (hunk 1 above)
Comment (jonatasalves-hotmart) [reply tests/test_utils.py]: there were spaces on the last line. already removed.

**Comment by jonatasalves-hotmart** [comment]
@Yury-Fridlyand Thank you for your review. Everything you requested has already been done.

**Review by Yury-Fridlyand — APPROVED** [review]

**Review by VachaShah — APPROVED** [review]

**Comment by jonatasalves-hotmart** [comment]
Thanks guys, is it possible to have a pre release already with this change?

**Comment by VachaShah** [comment]
Hi @jonatasalves-hotmart, this change will be released in the upcoming 2.1.0 version (related issue #98). Thank you!

**Pull request merged**